internal/server/my_details.go

```diff
@@ -10,6 +10,7 @@ import (
 
 type MyDetailsClient interface {
 	MyDetails(context.Context, []*http.Cookie) (sirius.MyDetails, error)
+	EditMyDetails(context.Context, []*http.Cookie, int, string) (sirius.ValidationErrors, error)
```

Comment:
My plan with these was that we'd have just the methods used by the handler, so you would have one in your new file like

```
type EditMyDetailsClient interface {
  MyDetailsClient
  EditMyDetails(context.Context, []*http.Cookie, int, string) (sirius.ValidationErrors, error)
}
```

Reply:
https://golang.org/doc/effective_go.html#embedding if that looks weird

Reply:
Yep, that makes perfect sense 👍